cassandra/src/cassandra/runner.clj

```diff
@@ -15,8 +15,8 @@
              [nemesis :as jn]]
             [jepsen.nemesis.time :as nt]))
 
-(def link-to-tarball "http://ftp.meisei-u.ac.jp/mirror/apache/dist/cassandra/3.11.4/apache-cassandra-3.11.4-bin.tar.gz")
-;(def link-to-tarball "http://www.us.apache.org/dist/cassandra/3.11.4/apache-cassandra-3.11.4-bin.tar.gz")
+(def link-to-tarball "http://ftp.meisei-u.ac.jp/mirror/apache/dist/cassandra/3.11.5/apache-cassandra-3.11.5-bin.tar.gz")
```

Comment:
I think we can use the official archive so that it won't fail even if there is some update.
https://archive.apache.org/dist/cassandra/

Reply:
Good idea! Thanks.